CMakeLists.txt

```diff
@@ -655,52 +665,68 @@ else()
 endif()
 
 if (MSVC)
-	# disable bogus deprecation warnings on msvc8
-	target_compile_definitions(torrent-rasterbar PUBLIC _SCL_SECURE_NO_DEPRECATE _CRT_SECURE_NO_DEPRECATE)
-	# these compiler settings just make the compiler standard conforming
-	target_compile_options(torrent-rasterbar PUBLIC /Zc:wchar_t /Zc:forScope)
-	# for multi-core compilation
-	target_compile_options(torrent-rasterbar PUBLIC /MP)
-	# increase the number of sections for obj files
-	target_compile_options(torrent-rasterbar PUBLIC /bigobj)
+	target_compile_definitions(torrent-rasterbar
+		PUBLIC
```

Comment:
These ones probably should be private?

Reply:
Eh, I think of it as a community service to disable that BS.